R/pkg/R/mllib.R

```diff
@@ -41,7 +41,8 @@ setClass("PipelineModel", representation(model = "jobj"))
 #' sqlContext <- sparkRSQL.init(sc)
 #' data(iris)
 #' df <- createDataFrame(sqlContext, iris)
-#' model <- glm(Sepal_Length ~ Sepal_Width, df)
+#' model <- glm(Sepal_Length ~ Sepal_Width, df, family=c("gaussian","binomial"))
```

Comment:
AFAIK we dont -- but shouldn't the family be either gaussian or binomial ? Or to put it another way the example should have separate lines for each family.